src/components/adminDashboard/components/AdminDashboard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Tag, Table, Icon, Input, Badge, Card, Col, Layout, Row } from 'antd';
+
+import { ALL_CAMPAIGNS, CARDS, NEW_CAMPAIGNS, VIDEO_CAMERA } from '../constants';
+import sideMenu from '../../sideMenu';
+
+const { SideMenu } = sideMenu.components;
+const { Content, Sider } = Layout;
+const { Search } = Input;
+const data = [
+    {
+        amount: '₦50000000',
+        customerName: 'Innoson',
+        date: 'May 26, 2019',
+        key: '1',
+        sn: '1',
+        status: ['live'],
+        transactionDetails: 'MTN Pulse Dec Promo',
+    },
+    {
+        amount: '₦20000000',
+        customerName: 'Aliko Dangote',
+        date: 'October 12, 2019',
+        key: '2',
+        sn: '2',
+        status: ['live'],
+        transactionDetails: 'MTN Hackathon with CCHub',
+    },
+    {
+        amount: '₦25000000',
+        customerName: 'Innoson',
+        date: 'November 15, 2019',
+        key: '3',
+        sn: '3',
+        status: ['expired'],
+        transactionDetails: 'Wizkid Concert with MTN',
+    },
+];
```

Comment:
`data` should be moved to constants file

Reply:
Alright